sdk/messaging/azeventhubs/consumer_client.go

```diff
@@ -22,19 +22,19 @@ import (
 // are subject to change.
 type Error = exported.Error
 
-// Code is an error code, usable by consuming code to work with
+// ErrorCode is an error code, usable by consuming code to work with
 // programatically.
-type Code = exported.Code
+type ErrorCode = exported.ErrorCode
 
 const (
-	// CodeConnectionLost means our connection was lost and all retry attempts failed.
+	// ErrorCodeConnectionLost means our connection was lost and all retry attempts failed.
 	// This typically reflects an extended outage or connection disruption and may
 	// require manual intervention.
-	CodeConnectionLost = exported.CodeConnectionLost
+	ErrorCodeConnectionLost = exported.ErrorCodeConnectionLost
```

Comment:
I've discovered that you need to include the type in the definition in order for doc generation to properly work.
```go
ErrorCodeConnectionLost ErrorCode = exported.ErrorCodeConnectionLost
```

Reply:
Done.